ratis-common/src/main/java/org/apache/ratis/protocol/RaftPeer.java

```diff
@@ -133,10 +136,24 @@ public Builder setPriority(int priority) {
       return this;
     }
 
+    public Builder setStartupRole(RaftProtos.RaftPeerRole startupRole) {
+      if (startupRole != RaftProtos.RaftPeerRole.FOLLOWER
```

Comment:
I think `!=` can be replaced by `!...equals` here

Reply:
Enume classes will have the same result when using `==` and `equals`.

Reply:
ok, ignore my personal habits